### ISSUE TITLE
A transferred conversation does not appear in the inbox of the assigned agent

### ISSUE THREAD

**Issue by MatthieuBietry** [chatwoot/chatwoot#1869]
**Describe the bug**

When I transfer (or assign) a conversation to a second agent, if this second agent does not already have an open conversation, the transfered conversation does not appear and an error occured. 
However, if it already has an open conversation, the operation works.

The problem seems to be when we update an assigned conversation on the action cable message, the `allConversations` present in the state is empty, and the `const [chat] = _state.allConversations.filter (c => c.id === payload.id );` therefore does not work.

**To Reproduce**

1. Go to Conversations (or an Inbox) with an agent 1, but without open an existing conversation
2. With an agent 2, transfer a conversation to agent 1
3. The conversation doesn't appear in the Agent 1 conversation list. 
4. If you refreh the page, or navigate in the tabs, the conversation appear. 

**Expected behavior**

A transfered conversation must be displayed in the agent conversation list, without need of refresh.

**Screenshots**

https://user-images.githubusercontent.com/4271987/110256176-c792ad80-7f97-11eb-876d-8bfe86edca54.mov

**Browser logs**

```
Uncaught TypeError: Cannot read property 'meta' of undefined
    at index.js:162
    at wrappedMutationHandler (vuex.js:771)
    at commitIterator (vuex.js:470)
    at Array.forEach (<anonymous>)
    at vuex.js:469
    at Store._withCommit (vuex.js:574)
    at Store.commit (vuex.js:468)
    at boundCommit (vuex.js:420)
    at updateAssignee (actions.js:202)
    at Array.wrappedActionHandler (vuex.js:778)
```

**Server logs**

No errors in the server logs

**Environment**

I observe this issue, on app.chatwoot.com, on an Heroku hosting or in my local server. 

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser : Chrome( 88.0) and Firefox (86.0)

**Comment by nithindavid** [comment]
This issue is already taken care in #1849. Will close this once that PR is merged and tested.

**Comment by nithindavid** [comment]
>  This issue is already taken care in #1849. Will close this once that PR is merged and tested. 
This isn't fixed with #1849, Will fix in a new PR.
